<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Showing the histogram of an image, convert to grayscale
• Enhance image quality using: histogram equalization, Gramma correction.
Filter the image with blurring, smoothing, sharpening.
• Saving an image to another formats (JPEG, PNG, BMP)
• Rotate, scaling, flip, warp, crop an image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="histogramShowBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Histogram</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imgshowBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="view_result">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="grayScaleShowBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Gray Scale</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
